src/families.h

```diff
@@ -47,68 +45,65 @@ class Family {
     step_sizes.reserve(alpha_scaled.size());
 
     for (auto alpha_val : alpha_scaled) {
-      double L = L_scaling*(max_squared_sum + fit_intercept) + alpha_val;
+      double L =
+        L_scaling_*(max_squared_sum + static_cast<double>(fit_intercept))
```

Comment:
Where is `fit_intercept` coming from here? It got dropped from the argument list.

Reply:
It is still there on line 41 I believe and is included in its called as this (in sgdnet.cpp): `family->StepSize(max_squared_sum, alpha, fit_intercept, n_samples);`

Reply:
Oops - my mistake. I didn't expand the "shrunk" part of the diff and thought this was the implementation of `PreprocessResponse` -- looks good.